### ISSUE TITLE
Examples fail - cannot find modules.

### ISSUE THREAD

**Issue by SolebaySharp** [kazhala/InquirerPy#36]
Trying to run examples to see if I can switch to a maintained port of inquirer.js (desperately need it). But I can't run example scripts.

`from InquirerPy.base.control import Choice`
`ModuleNotFoundError: No module named 'InquirerPy.base.control'; 'InquirerPy.base' is not a package`

**Comment by kazhala** [comment]
Hey @SolebaySharp ,

Could you show a bit more information on which examples you are trying to run? I just did a test run from the root of the project and it's working fine.

```sh
source venv/bin/activate
python examples/classic/rawlist.py
```

Thanks

**Comment by SolebaySharp** [comment]
Stand by, I think I've been an idiot.

**Comment by SolebaySharp** [comment]
Yeah, Python reverted to an old version not compatible with InquierePy. Updating past Python3.6 prevented gnome terminal from working. It appears Python3.10 prevented InquirerPy and Terminal from working. Reverting to 3.9 allowed InquiererPy to work but changes in my virtual environment were going global, interfering with gnome.

At some point while I was trying to work out what was going on, I think I purged half my desktop and so I spent the day reinstalling my OS.

Examples are working just fine now. Thank you for your rapid response.
